web/impact/impact/tests/test_graphql.py

```diff
@@ -582,8 +562,13 @@ def test_query_for_user_without_confirmed_mentor_program_families(self):
             self.assertEqual(expert_profile["confirmedMentorProgramFamilies"],
                              [])
 
-    def _assert_response_equals_json(self, query, expected_json):
-        with self.login(email=self.basic_user().email):
+    def _assert_response_equals_json(self, query, expected_json,
```

Comment:
if we have to break lines for any arguments, let's put each on its own line for readability